src/lib/utils.ts

```diff
@@ -64,3 +64,5 @@ export const isValidAccountNumber = (
   if (accountNumber === null) return false
   return Number.isInteger(accountNumber) && accountNumber >= 0
 }
+
+export type PartialRecord<K extends keyof any, T> = Partial<Record<K, T>>
```

Comment:
Absolute nitpick but what's `T` here ?
I believe we should either use`K, V` generics (Key, value), or the conventional ordered `T, U`

Reply:
yeah strictly a record is only keyed by `string | number | symbol`

```
> type R = Record<unknown, unknown>
<repl>.ts:4:17 - error TS2344: Type 'unknown' does not satisfy the constraint 'string | number | symbol'.

4 type R = Record<unknown, unknown>
```

Reply:
nvm `keyof any` *is*` `string | number | symbol` 🤦 

```
➜  web git:(develop) npx ts-node --skipProject
> type K = keyof any
undefined
> const k: K = function() {}
<repl>.ts:5:7 - error TS2322: Type '() => void' is not assignable to type 'string | number | symbol'.

5 const k: K = function() {}
```

Reply:
but the amount i wish i did this 12 months ago...

Reply:
> but the amount i wish i did this 12 months ago...

We can fix this 😄

Reply:
> Absolute nitpick but what's `T` here ? I believe we should either use`K, V` generics (Key, Value), or the conventional ordered `T, U`

Yeah, you're right there gomes - this should probably be:

```ts
export type PartialRecord<K extends keyof any, V> = Partial<Record<K, V>>
```